doc/cs_registers.rst

```diff
@@ -483,6 +485,26 @@ Reset Value: ``0x0000_0000``
 Scratch register to be used by the debug module.
 Accessible in Debug Mode only.
 
+CPU Control Register (cpuctrl)
+------------------------------------
+
+CSR Address: ``0x7C0``
```

Comment:
Is this address in an designated extension space, or could we clash in the future with the RISC-V spec?

Reply:
It's the bottom of one of the designated "custom" read write machine mode spaces. Wasn't really sure where to put it since we don't have anything similar already in Ibex.